libc/string/string_tokpbrkstr.c

```diff
@@ -736,6 +736,10 @@ TEST(string_str, basic)
 
 TEST(string_str, empty_args)
 {
+	/* Ifdef used because of wrong implementation */
+#ifdef __phoenix__
+	TEST_IGNORE();
+#else
```

Comment:
Implementation has been fixed in https://github.com/phoenix-rtos/libphoenix/commit/173d3fadca6d1df64cbc65de2c1593ef6671a667
Just submodules not updated yet - I'll update.